dist/src/contracts/Iglobal.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Window {
+    signalRInvocationResults: any;
+}
```

Comment:
Should not be gitignored the whole dist folder?

Reply:
Yes, you are right. The whole `dist` folder could be auto generated by running `tsc`.
I commited it because:

- I followed this repo's pattern(existing files in `dist`) to commit `dist`.
- The entry point is `dist/index.js`
- User could use the lib without build.

But if you think it is unnecessary, please remove.